INSTRUCTION.md

```diff
@@ -0,0 +1,13 @@
+# Instructions for ToDo App
+
+## Docker Hub Repository
+The Docker image for this application is available on Docker Hub:
+[https://hub.docker.com/r/yur11lesyk/devops-todolist](https://hub.docker.com/r/yur11lesyk/devops-todolist)
+
+Replace `<your-dockerhub-username>` with your actual Docker Hub username.
```

Comment:
The instruction to replace `<your-dockerhub-username>` with your actual Docker Hub username seems unnecessary here, as the Docker Hub link provided already contains a specific username. Ensure that users understand whether they need to modify the link or if this is a placeholder for their own repository setup.